client/src/components/Explorer/ExplorerPanel.js

```diff
@@ -6,7 +6,7 @@ import { STRINGS, MAX_EXPLORER_PAGES } from '../../config/wagtailConfig';
 
 import Button from '../Button/Button';
 import LoadingSpinner from '../LoadingSpinner/LoadingSpinner';
-import Transition, { PUSH, POP, FADE } from '../Transition/Transition';
```

Comment:
If we are not using `FADE` at the moment, we can clean this up from `Transition.js` and `Transition.scss` as well for now.